configure.ac

```diff
@@ -1158,12 +1158,6 @@ AS_IF([test "x$with_openssl" = "xyes"],[
     ],$LIBOPENSSL_LIBS)
   ])
 
-  # This is a workaround for RedHat 9 brain damage..
```

Comment:
RedHat 9 was released on 2022-05-17. I assume this 2003 comment has a RedHat version typo.

I am curious how do you know that these RedHat workarounds are no longer needed (on still supported RedHat flavors).

Reply:
Original `RedHat 9`, not part of the `Enterprise Linux` series.